src/Features/ERDDAP/Platform/Forecasts/index.tsx

```diff
@@ -1 +1,172 @@
+ */
+export const Forecast: React.FunctionComponent<Props> = ({ platform, forecast_type, ...props }) => {
+  const standardNames = forecastToStandardNames[forecast_type]
+  const timeSeries = platform.properties.readings.find((r) => standardNames.has(r.data_type.standard_name))
+
+  return <LoadForecastInfo {...props} {...{ platform, forecast_type, timeSeries }} />
+}
+
+interface LoadInfoProps extends Props {
+  timeSeries?: PlatformTimeSeries
+}
+
+/**
+ * Load all the forecasts, and load a matching time series if available.
+ */
+const LoadForecastInfo: React.FunctionComponent<LoadInfoProps> = ({ timeSeries, forecast_type, ...props }) => {
+  const { isLoading: isLoadingForecasts, data: forecastsInfo } = useForecasts()
+  const { data: dataset } = useDataset(timeSeries)
+
+  const forecastInfo = forecastsInfo?.find((f) => f.forecast_type.toLowerCase().replace(/ /g, "_") === forecast_type)
+
+  if (isLoadingForecasts) {
+    return <Alert>Loading forecast metadata</Alert>
+  }
+
+  if (forecastInfo) {
+    return <LoadForecast {...props} {...{ timeSeries, dataset, forecast_type, forecastInfo }} />
+  }
+
+  return <Alert color="warning">Error loading forecast info</Alert>
+}
+
+interface LoadForecastProps extends LoadInfoProps {
+  forecastInfo: ForecastSource
+  dataset?: DataTimeSeries
+}
+
+/**
+ * Load the forecast itself
+ */
+const LoadForecast: React.FunctionComponent<LoadForecastProps> = ({
+  forecastInfo,
+  platform,
+  unitSystem,
+  timeSeries,
+  dataset,
+  forecast_type,
+}) => {
+  const point = platform.geometry as Point
+  const [lon, lat] = point.coordinates
+
+  const { isLoading, data } = useForecast(lat, lon, forecastInfo)
+
+  if (isLoading) {
+    return <Alert>Loading forecast</Alert>
+  }
+
+  if (data) {
+    const forecastDataset: DataTimeSeries = {
+      timeSeries: data,
+      name: forecastInfo.name,
+      unit: forecastInfo.units,
+    }
+
+    const datasets = [forecastDataset]
+
+    if (timeSeries && dataset) {
+      const aDayAgo = aDayAgoRounded()
+
+      datasets.push({
+        ...dataset,
+        timeSeries: dataset.timeSeries.filter((r) => aDayAgo < r.time),
+        name: `${timeSeries.dataset}: ${timeSeries.data_type.long_name}`,
+      })
+    }
+
+    return (
+      <Row>
+        <Col>
+          <h4>{forecastInfo.forecast_type} Forecast</h4>
+          <ForecastChart data={datasets} unitSystem={unitSystem} type={forecast_type} />
+        </Col>
+      </Row>
+    )
+  }
+
+  return <Alert color="warning">Unable to load forecasts</Alert>
+}
+
+/**
+ * Which forecasts match up with which standard names
+ */
+export const forecastToStandardNames: { [key: string]: Set<string> } = {
+  air_temperature: new Set(["air_temperature"]),
+  significant_wave_height: new Set([
+    "sea_surface_wave_significant_height",
+    "significant_height_of_wind_and_swell_waves",
+    "significant_wave_height",
+  ]),
+  wave_direction: new Set(["sea_surface_wave_to_direction"]),
```

Comment:
Codacy found an issue: [Identifier 'wave_direction' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6288046)